tracker/enqueue_test.go

```diff
@@ -58,7 +58,7 @@ func TestHappyPathsExact(t *testing.T) {
 
 	thing2 := &Resource{
 		TypeMeta: metav1.TypeMeta{
-			APIVersion: "reffer.knative.dev/v1alpha1",
+			APIVersion: "refer.knative.dev/v1alpha1",
```

Comment:
very worried about anything with the word `api`, but I'm pretty sure this is just a test.